backstop/tests/menu.html

```diff
@@ -320,6 +320,53 @@ <h2>Default</h2>
         </li>
       </ul>
 
+      <br />
+
+      <ul class="iui-menu">
+        <li class="iui-menu-content">
+          <div class="iui-text-leading">Terry Rivers</div>
+          <div
+            class="iui-text-muted"
+            style="margin-bottom: 8px;"
+          >terry.rivers@email.com</div>
+          <label class="iui-input-container">
+            <div class="iui-select">
+              <div
+                tabindex="0"
+                class="iui-select-button"
+              >
+                <span class="iui-content">Administrator</span>
+              </div>
+            </div>
+          </label>
+
+        </li>
+        <li
+          class="iui-menu-divider"
+          aria-hidden="true"
```

Comment:
Can you change `aria-hidden="true"` to `role="separator"` in every divider? Because it is the preferable way from accessibility point.

Reply:
Yes, I can do that, good suggestion.  @mayank99 was suggesting that we also apply some sort of aria on the new `.iui-menu-content` since it's not selectable like the rest of the menu items.  Any suggestions?

Reply:
Best I can think of is `role='presentation'` but I'm not sure

Reply:
I've added `role="presentation"` as well as some other roles and aria attributes found in [this article](https://www.smashingmagazine.com/2017/11/building-accessible-menu-systems/).